deepmd/entrypoints/test.py

```
@@ -243,6 +246,14 @@ def save_txt_file(
         np.savetxt(fp, data, header=header)
 
 
+def whether_hessian(dp: "DeepPot"):
+    # If model type is EnergyHessianModel, return True and print hessian loss info
+    if "Hessian" in str(type(get_model(dp.deep_eval.input_param))):
+        return True
+    else:
+        return False
```

Ruff

Return the condition directly

Inline condition

(SIM103)